### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by adagostino09**
@joewadcan Checking my name for pull request assignment!

### CODE REVIEW

**Comment by joewadcan** [comment]
👋  @adagostino09 Looks like there's a type with your 🍍 emoji - can you edit the file again and commit a new change to this branch?

File: 2018_Fall/readme.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Make the Pull Request from your own GitHub account. Make sure you're changing th
 - [ ] Cornejo, Ricky
 - [ ] Crestani, Ryan
 - [ ] Czymoch, Maximilian
-- [ ] D'Agostino, Alex
+- [X] D'Agostino, Alex:pinneaple::boom:
```
Comment (joewadcan) [inline 2018_Fall/readme.md]: I think it's `:pineapple:`

Comment (adagostino09) [reply 2018_Fall/readme.md]: Thanks Joe! I think I edited it - let me know how it looks now!

File: 2018_Fall/readme.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Make the Pull Request from your own GitHub account. Make sure you're changing th
 - [ ] Cornejo, Ricky
 - [ ] Crestani, Ryan
 - [ ] Czymoch, Maximilian
-- [X] D'Agostino, Alex:pinneaple::boom:
+- [X] D'Agostino, Alex:pinneaple:
```
Comment (joewadcan) [inline 2018_Fall/readme.md]: so close, but `pineapple` is still misspelled. IF you see the View button you can see if it's working

**Comment by joewadcan** [comment]
One more try @adagostino09 !

**Comment by joewadcan** [comment]
Great! That's @adagostino09

**Pull request merged**